chart/tests/test_airflow_common.py

```diff
@@ -248,3 +248,22 @@ def test_have_all_variables(self):
             assert variables == jmespath.search(
                 "spec.template.spec.containers[0].env[*].name", doc
             ), f"Wrong vars in {component}"
+
+    def test_have_all_config_mounts_on_init_containers(self, expected_read_only):
+        docs = render_chart(
+            values={},
+            show_only=[
+                "templates/scheduler/scheduler-deployment.yaml",
+                "templates/workers/worker-deployment.yaml",
+                "templates/webserver/webserver-deployment.yaml",
+                "templates/triggerer/triggerer-deployment.yaml",
+            ],
+        )
+        assert 4 == len(docs)
+        for doc in docs:
+            expected_mount = {
+                "subPath": "airflow.cfg",
+                "name": "config",
+                "readOnly": expected_read_only,
+            }
```

Comment:
```suggestion
        expected_mount = {
            "subPath": "airflow.cfg",
            "name": "config",
            "readOnly": True,
            "mountPath": "something",
        }
        for doc in docs:
```

We aren't passing in `expected_read_only`, and we don't need to set this in every loop. We do need to set the mountPath (not sure what it would be off the top of my head).